learning-bitmark/working-with-bitmarks/transferring-bitmarks.md

```diff
@@ -0,0 +1,358 @@
+      "payments": {
+        "BTC": [
+          {
+            "currency": "BTC",
+            "address": "mr8DEygRvQwKfP4sVZuHVozqvzW89e193j",
+            "amount": "20000"
+          }
+        ],
+        "LTC": [
+          {
+            "currency": "LTC",
+            "address": "mzkCaHJmu1gdnsL9jxW2bwqtw2MCCy66Ds",
+            "amount": "200000"
+          }
+        ]
+      }
+    }
+    ```
+
+<br>
+
+* Paying for the bitmark transfer using the Bitmark-Wallet
+
+    ```shell
+    #Run the bitcoind
+    $ bitcoind -datadir=<bitcoind config dir>
+
+    # OR run the litecoind
+    $ litecoind -datadir=<litcoind config dir>
+
+    #Pay by BTC
+    $ bitmark-wallet --conf <Bitmark-Wallet config file> btc --<btc network> \
+      sendmany --hex-data '<payId>' '<btc address>,<btc amount in satoshi>'
+
+    #OR Pay by LTC
+    $ bitmark-wallet --conf <Bitmark-Wallet config file> ltc --<ltc network> \
+      sendmany --hex-data '<payId>' '<ltc address>,<ltc amount in photon>'
+
+    ```
+
+    >To execute a bitcoin or litecoin transaction on the local environment, the bitcoind or litecoind need to be running.
+    >
+    > - `payId` - The payment id for the share creation transaction, it is printed in the output of share creation command
+
+    <br>
+
+    > **NOTE:** The payment is able to be performed using another tool which allows users to add the exact hex data to an OP_RETURN as part of the coin transaction
+
+    *Example of paying by ltc:*
+
+    ```shell
+    # run litecoind
+    $ bitcoind -datadir=${HOME}/.config/litecoin/
+
+    # Perform payment
+    $ bitmark-wallet --conf ${XDG_CONFIG_HOME}/bitmark-wallet/test/test-bitmark-wallet.conf \
+      ltc --testnet \
+      sendmany --hex-data \
+      'd819cff364b9211093fe09c2b462bdd05154472a72fac91a882a8f1129674dc92ac5d2724c8d26b16d414de8fbc5c62e' \
+      'mzkCaHJmu1gdnsL9jxW2bwqtw2MCCy66Ds,200000'
+    ```
+    ```json
+    {
+      "txId": "ca94ae188ba8bfdc42e026950c5e13a2f1082dae484a45c5dc29217ac0c9a23f",
+      "rawTx": "0100000001b76a37054a086c5bd68afd61914bb4badc78c9e7ef59e6b692777cc18063632d020000006b483045022100db6f27ec3e1e59c34887f262217d5ff819947c561f0ecd11034ba8b32dbdc87002203ef82d80be8e43434eb16e22248e4533a1d3c2831e19802ce55a308604d76f3c012103b45a55c3e48209581d63ba5ceea9a0e94ae49e18056d85a6dadec535dbe237a2ffffffff03400d0300000000001976a914d2ebb7b259fb7410dca19b707c4091195d818ac488ac8091e305000000001976a9142d477753d17099534f9249b54cda36081d4e5eba88ac0000000000000000326a30d819cff364b9211093fe09c2b462bdd05154472a72fac91a882a8f1129674dc92ac5d2724c8d26b16d414de8fbc5c62e00000000"
+    }
+    ```
+
+<br>
+
+* Verifying the status of the share creation transaction
+
+    ```shell
+    $ bitmark-cli -n <network> \
+      status -t <txid>
+    ```
+
+    >The `status` command is to query a transaction's status. The options:
+    >
+    >* `txid` - The transaction id which is printed in the output of the `transfer` command.
+    >
```

Comment:
I'm not sure which id you're talking about. I think you mean the Bitmark `transferid` from the `transfer` command. But, whichever it is, it should be stated very clearly.

Reply:
updated.